Client/src/Utils/Theme/constants.js

```diff
@@ -0,0 +1,244 @@
+const typographyBase = 14;
+
+/* TODO
+Check for px in codebase. All font sizes should be in REM and should live here.
+Rest should be checked on each case
+*/
+const typographyLevels = {
+	base: typographyBase,
+	xs: `${(typographyBase - 4) / 16}rem`,
+	s: `${(typographyBase - 2) / 16}rem`,
+	m: `${typographyBase / 16}rem`,
+	l: `${(typographyBase + 2) / 16}rem`,
+	xl: `${(typographyBase + 10) / 16}rem`,
+};
+
+/* TODO Review color palette and semantic colors */
+const paletteColors = {
+	white: "#FFFFFF",
+	gray50: "#FEFEFE",
+	gray60: "#FEFDFE",
+	gray70: "#FDFDFD",
+	gray80: "#FDFCFD",
+	gray90: "#FCFCFD",
+	gray100: "#F4F4F4",
+	gray200: "#E3E3E3",
+	gray300: "#A2A3A3",
+	gray500: "#838C99",
+	gray600: "#454546",
+	gray750: "#36363E",
+	gray800: "#2D2D33",
+	gray850: "#131315",
+	gray860: "#111113",
+	gray870: "#0F0F11",
+	gray880: "#0C0C0E",
+	gray890: "#09090B",
+	blueGray20: "#E8F0FE",
+	blueGray150: "#667085",
+	blueGray200: "#475467",
+	blueGray400: "#344054",
+	blueGray900: "#1c2130",
+	blueBlueWave: "#1570EF",
+	blue700: "#4E5BA6",
+	purple300: "#664EFF",
+	purple400: "#3A1BFF",
+	green50: "#D4F4E1",
+	green150: "#45BB7A",
+	green400: "#079455",
+	green800: "#1C4428",
+	green900: "#12261E",
+	red50: "#F9ECED",
+	red100: "#FBD1D1",
+	red200: "#F04438",
+	red300: "#D32F2F",
+	red700: "#542426",
+	red800: "#301A1F",
+	orange50: "#FEF8EA",
+	orange100: "#FFECBC",
+	orange300: "#FDB022",
+	orange400: "#FF9F00",
+	orange500: "#E88C30",
+	orange600: "#DC6803",
+	orange800: "#624711",
+};
+
+const semanticColors = {
+	primary: {
+		main: {
+			light: paletteColors.blueBlueWave,
+			dark: paletteColors.blueBlueWave,
+		},
+		// TODO we dont have primary light, dark or contrast text
+	},
+	secondary: {
+		main: {
+			light: paletteColors.gray100,
+			dark: paletteColors.gray800,
+		},
+		constrastText: {
```

Comment:
Typo: constrastText -> contrastText

Reply:
Wow, thanks for catching that!

Reply:
Done here https://github.com/bluewave-labs/bluewave-uptime/pull/1152/commits/b13f4b1633d8a3bb9fe078b7e3545b34a772a3d2